engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -23,7 +23,7 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
 
-  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson], workflowOptions: Option[WdlJson]): Props = {
+  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson] = None, workflowOptions: Option[WdlJson] = None): Props = {
```

Comment:
I agree, my point is as long as it doesn't add unnecessary internal mutable data management / maintenance to the worker actor then I don't really mind if the data is in the message.

One example of something nice to have as a parameter is the actor to send the response to. If you include it in the message, then you have to set a variable somewhere (in the actor data probably), or keep passing it in every single message until you're finally ready to send your response. If you have it as a parameter it's available from the very beginning to the end, for no cost.